MonMaraicher/Features/Map/FarmerServiceMock.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  FarmerServiceMock.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 07/05/2024.
+//
+
+import Foundation
+import CoreLocation
+
+class FarmerServiceMock: FarmerServiceProtocol {
```

Comment:
Did you add that file only for the test target? Your main target shouldn't have it

Reply:
I didn't understand the question